<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cellClass</class>
 <widget class="QMainWindow" name="cellClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>232</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>563</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>cell</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="titlePlace">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="deleteTask">
            <property name="text">
             <string>delete task</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="displayImage">
            <property name="text">
             <string>display </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="update">
            <property name="text">
             <string>update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="showInfo">
            <property name="text">
             <string>showInfo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="upload">
            <property name="text">
             <string>upload</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
